src/adapters/triplelift.js

```diff
@@ -31,21 +31,24 @@ var TripleLiftAdapter = function TripleLiftAdapter() {
   function buildTLCall(bid, callbackId) {
     //determine tag params
     var inventoryCode = utils.getBidIdParamater('inventoryCode', bid.params);
+    var floor = utils.getBidIdParamater('floor', bid.params);
+
 
     //build our base tag, based on if we are http or https
     var tlURI = '//tlx.3lift.com/header/auction?';
     var tlCall = document.location.protocol + tlURI;
 
     tlCall = utils.tryAppendQueryString(tlCall, 'callback', 'pbjs.TLCB');
     tlCall = utils.tryAppendQueryString(tlCall, 'lib', 'prebid');
-    tlCall = utils.tryAppendQueryString(tlCall, 'lib', '0.5.0');
+    tlCall = utils.tryAppendQueryString(tlCall, 'v', '0.5.0');
```

Comment:
Is the version here for TripleLift or prebid.js? If prebid, then you can use the `$prebid.version$` macro to populate from the build process.